File: BACK-END/src/main/java/org/kuenteco/backend/dto/account/NewAccountDTO.java
package org.kuenteco.backend.dto.account;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.kuenteco.backend.enums.AccountType;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class NewAccountDTO {
    private String name;
    private AccountType type;
}

File: FRONT-END/lib/domain/dto/New_Account_DTO.dart
import 'package:kuenteco/domain/dto/Account_type.dart';

class NewAccountDTO {
  final String name;
  final AccountType type;

  NewAccountDTO({
    required this.name,
    required this.type,
  });

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'type': type.name,
    };
  }
}
